test/model/content_test.dart

```diff
@@ -615,6 +699,233 @@ class ContentExample {
       ]),
     ]);
 
+  // The font sizes can be compared using the katex.css generated
+  // from katex.scss :
+  //   https://unpkg.com/katex@0.16.21/dist/katex.css
+  static final mathBlockKatexSizing = ContentExample(
+    'math block; KaTeX different sizing',
+    // https://chat.zulip.org/#narrow/channel/7-test-here/topic/Rajesh/near/2147558
+    "```math\n\\tiny {\\lambda \\Huge \\lambda}\n\n\\Huge \\lambda\n\\huge \\lambda\n\\LARGE \\lambda\n\\Large \\lambda\n\\large \\lambda\n\\normalsize \\lambda\n\\small \\lambda\n\\footnotesize \\lambda\n\\scriptsize \\lambda\n\\tiny \\lambda\n```",
```

Comment:
Instead of `\lambda` here, how about just `x`? That'd make this shorter and easier to read, given how many copies of it this has.

Some other test cases say `\lambda` because that's a way of making the rendered output have different text from the input, so that it's easy to check which is being displayed. But I think those test cases cover that aspect, so this one doesn't need that wrinkle.

Reply:
Or instead of "x x x x" could be like "a b c d" etc., if that helps make things convenient for inspecting in a widget test.